site-builder/src/site/resource.rs

```diff
@@ -323,13 +368,16 @@ pub(crate) struct ResourceManager {
     pub walrus: Walrus,
     /// The resources in the site.
     pub resources: ResourceSet,
+    /// The ws-config.json contents.
+    pub ws_config: Option<WSConfig>,
```

Comment:
I defined this as optional because it is not required to specify a `ws_config` for every walrus site. If it's not an option, then ws_config should always be contained in the user's folder right?  

Having optional fields on the other hands, implies that the file is defined, but some of the fields are missing.